modules/Bio/EnsEMBL/Utils/Interval.pm

```diff
@@ -159,7 +193,11 @@ sub data {
 sub is_empty {
   my $self = shift;
 
-  return $self->start >= $self->end;
+  if ($self->spans_origin) {
```

Comment:
By definition of `spans_origin`, we have `$start > $end` in the_then_ block, so it will always return 0. And in the _else_ block `$start <= $end`, so it will only return 1 if `$start -- $end`, i.e. if the interval is a "point"